connect/utils.py

```diff
@@ -45,3 +45,21 @@ def count_contacts(project: Project, before: str, after: str):
         created_at__gte=after
     )
     return sum([day_count.count for day_count in contacts_day_count])
+
+
+def check_module_permission(claims, user):
+    from django.contrib.auth.models import Permission
```

Comment:
Is there any specific reason why the imports are inside the function?

Reply:
It's because in utils the function is invoked separately, so the import is for the specific function, isn't that the recommendation?